src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -57,7 +57,9 @@
       <nav class="hmcts-primary-navigation transparent-background" aria-label="Primary navigation">
         <ul class="hmcts-primary-navigation__list">
           <li class="hmcts-primary-navigation__item" *ngFor="let item of items" >
-            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item.active ? true: null" [routerLink]="item.href">{{item.text}}</a>
+            
+            <a class="hmcts-primary-navigation__link" (click)="onEmitSubMenu(item)" [attr.aria-current]="item === selectedItem || setTab === item.text ? true: null" [routerLink]="item.href">{{item.text}}</a>
```

Comment:
(item === selectedItem || setTab === item.text) ? true: null

Reply:
Edited this - no longer need selectedItem